README.md

```diff
@@ -0,0 +1,15 @@
+# Fantasy Fitness
+
+## Development
+
+For development run the client and server on two different applications.
+
+To run the client application in development, cd into the client folder and `npm start`. 
```

Comment:
For the client app, if you don't specify to start the app in PORT 5000, the app creates this error:
Unhandled Rejection (SyntaxError): Unexpected token P in JSON at position 0
(anonymous function)
C:/Users/Darren/Desktop/FitnessApp/FantasyFitness-1.0/client/src/App.js:30
  27 | 
  28 | componentDidMount() {
  29 | 	fetch('/api')
> 30 | 		.then(res => res.json())
  31 | 		.then(res => console.log(res));
  32 | 
  33 | 	// Exercise array with inital value of each exercise

The terminal also shows this error:
Proxy error: Could not proxy request /api from localhost:3000 to http://localhost:5000/.
See https://nodejs.org/api/errors.html#errors_common_system_errors for more information (ECONNREFUSED).

Seems like the proxy is not working.

Reply:
So the proxy port has to match whatever is the server port. By default, it will use 5001, if no $PORT environment variable is specified. And proxy is only used in development.

On heroku, the server port number will not be 5001 but it will have a $PORT environment variable. The server code already accounts for this.